### PR TITLE
Refactors graph functionality (PR 3)

### PR DESCRIPTION
**Pull request by knod**
I _think_ it's not broken on this commit.

### CODE REVIEW

File: src/utils/convert-by-timescale.js [hunk 1]
```diff
@@ -70,9 +70,30 @@ var toWeeklyFrom = function (amount, startTimescale) {
...
+timescaleMultipliers.fromMonthly = {
+  'Weekly':  1 / (4 + 1 / 3),
+  'Monthly': 1,
+  'Yearly':  12,
+};
+
+timescaleMultipliers.fromWeekly = {
+  'Weekly':  1,
+  'Monthly': (4 + 1 / 3),
+  'Yearly':  12,
```
Comment (ethanbb) [inline src/utils/convert-by-timescale.js]: This should be `12 * (4 + 1 / 3)` (or just 52).

Comment (knod) [reply src/utils/convert-by-timescale.js]: Well shucks. Yeah, you're right. It's not exactly 52 - it's 52.14 and change, but maybe that's the way they do it anyway.

Comment (ethanbb) [reply src/utils/convert-by-timescale.js]: 12 * (4 + 1/3) does equal 52 though...

Comment (knod) [reply src/utils/convert-by-timescale.js]: Well, then I'm probably hallucinating. I got that number somehow, but not sure how ¯\_(ツ)_/¯

**Pull request merged**